packages/docprovider/src/users-item.tsx

```diff
@@ -0,0 +1,192 @@
+/*
```

Comment:
Is docprovider the right place for this, or should be in the `collaboration` package?

Reply:
I wondered the same, and I put it here because it is really a document related widget, it supposed to handle the document current collaborators.
But I don't have a strong opinion on it, and can move it to `collaboration`.